frontend/src/Components/Events/CustomizableEventConsole.svelte

```diff
@@ -199,6 +222,7 @@
                     }}>
                     RSVP List
                 </button>
+                <button on:click={() => copyToClipboard(emailsCheckedOff)}>Copy Email</button>
```

Comment:
Small issue, add an s to Email -> "Copy Emails".

Reply:
Fixed.